src/fluree/db/query/block.cljc

```diff
@@ -22,12 +30,10 @@
             res         (<? (storage/block conn network dbid next-block))
             root?       (-> db :permissions :root?)
             ;; Note this bypasses all permissions in CLJS for now!
-            res #?(:cljs res                                ;; always allow for now
-                   :clj (if root?
-                          res
-                          {:block  (:block res)
-                           :t      (:t res)
-                           :flakes (<? (perm-validate/allow-flakes? db (:flakes res)))}))
+            res #?(:cljs (if (identical? "nodejs" cljs.core/*target*)
+                           (filter-block-flakes db res)
+                           res)  ;; browser: always allow for now
+                   :clj (filter-block-flakes db res))
```

Comment:
This as it is written is async, and would need a <? (after wrapping it in a go-block)